nym-browser-extension/storage/Makefile

```diff
@@ -1,2 +1,2 @@
 wasm-pack:
-	wasm-pack build --scope nymproject #--target no-modules
\ No newline at end of file
+	wasm-pack build
```

Comment:
why did we remove the `nymproject` scope?

Reply:
I wasn't sure if it was necessary? I can put it back if it's required, but everything seems to be working without it?

Reply:
I think this would add the `@nymproject` at the beginning of import paths so it's explicit its ours. Whether we need it or not, I'm not sure. Perhaps @mmsinclair would have an opinion.

Reply:
Ah OK, I see. I'll run it past Mark. I think it makes sense to have it if it's likely to get consumed in other packages.